config/webpack.config.js

```diff
@@ -378,7 +378,7 @@ module.exports = async (env = {}) => {
             warnings: false,
           },
           // cache: true,
-          parallel: 3,
+          parallel: true,
```

Comment:
- By default, `parallel` is already set to `true`. 
- Is it faster to have it set to `true` than to `3` multi-processes?

Reply:
It is `true` by default. I left it in the config so that parallelization is explicit. `parallel: true` uses `os.cpus().length - 1` as the number of parallel processes, which seems preferred over using a fixed number.